flocker/provision/_install.py

```diff
@@ -241,9 +241,9 @@ def install_cli_commands_ubuntu(distribution, package_source):
         # Minimal images often have cleared apt caches and are missing
         # packages that are common in a typical release.  These commands
         # ensure that we start from a good base system with the required
-        # capabilities, particularly that the add-apt-repository command
-        # and HTTPS URLs are supported.
-        # FLOC-1880 will ensure these are necessary and sufficient.
+        # capabilities, particularly that sudo and add-apt-repository
+        # commands are available, and HTTPS URLs are supported.
+        run("which sudo || su root -c 'apt-get -y install sudo'"),
```

Comment:
There is a comment on https://clusterhq.atlassian.net/browse/FLOC-1880 which suggests that that issue will be covered, for the client, by this issue. However this adds to the list of necessary commands which are not in the docs for the people using those minimal images. Either improve how this deals with the requirements of FLOC-1880 (commands should be in the docs, docker images tested should be minimal) or update that issue accordingly to mention this series of commands.